src/facade/reply_payload.h

```diff
@@ -19,12 +23,16 @@ using Null = std::nullptr_t;  // SendNull or SendNullArray
 struct CollectionPayload;
 struct SimpleString : public std::string {};  // SendSimpleString
 struct BulkString : public std::string {};    // SendBulkString
-struct StoredReply {
-  bool ok;  // true for SendStored, false for SendSetSkipped
-};
+
+using ReplyFunction =
+    fu2::function_base<true /*owns*/, false /*non-copyable*/, fu2::capacity_fixed<16, 8>,
+                       false /* non-throwing*/, true /* strong exceptions guarantees*/,
+                       void(SinkReplyBuilder*)>;
```

Comment:
CollectionPayload is quite large, maybe you can give it more stack space

Reply:
we store std::unique_ptr<CollectionPayload> which is just 8 bytes